mkosi/sandbox.py

```diff
@@ -100,17 +100,37 @@ class cap_user_data_t(ctypes.Structure):
 libc.fcntl.argtypes = (ctypes.c_int, ctypes.c_int, ctypes.c_int)
 
 
-ENOSYS_MSG = """
-mkosi was unable to invoke the {syscall}() system call.
+def terminal_is_dumb() -> bool:
+    if not sys.stdout.isatty() and not sys.stderr.isatty():
+        return True
 
+    return os.getenv("TERM", "") == "dumb"
+
+
+class Style:
+    # fmt: off
+    bold: str   = "\033[0;1;39m"     if not terminal_is_dumb() else ""
+    blue: str   = "\033[0;1;34m"     if not terminal_is_dumb() else ""
+    gray: str   = "\033[0;38;5;245m" if not terminal_is_dumb() else ""
+    red: str    = "\033[31;1m"       if not terminal_is_dumb() else ""
+    yellow: str = "\033[33;1m"       if not terminal_is_dumb() else ""
+    reset: str  = "\033[0m"          if not terminal_is_dumb() else ""
+    # fmt: on
+
+
+ENOSYS_MSG = """\
+{red}mkosi was unable to invoke the {syscall}() system call.{reset}
 This probably means either the system call is not implemented by the running kernel version ({kver}) or the
-system call is prohibited via seccomp if mkosi is being executed inside a containerized environment.
+system call is prohibited via seccomp if mkosi is being executed inside a containerized environment.\
 """
```

Comment:
This could be formatted here (like below)
```python
ENOSYS_MSG = f"""\
{Style.red}mkosi was unable to invoke the {{syscall}}() system call.{Style.reset}
This probably means either the system call is not implemented by the running kernel version ({{kver}}) or the
system call is prohibited via seccomp if mkosi is being executed inside a containerized environment.\
"""